content/en/docs/setup/best-practices/certificates.md

```diff
@@ -135,9 +137,14 @@ You must manually configure these administrator account and service accounts:
 | kubelet.conf            | default-auth               | system:node:`<nodeName>` (see note) | system:nodes   |
 | controller-manager.conf | default-controller-manager | system:kube-controller-manager |                |
 | scheduler.conf          | default-scheduler          | system:kube-scheduler          |                |
+| proxier.conf            | default-proxier            | system:kube-proxy              |                |
+
+[3]: any IP or DNS name assigned to your kubelet/kube-proxy host.
 
 {{< note >}}
 The value of `<nodeName>` for `kubelet.conf` **must** match precisely the value of the node name provided by the kubelet as it registers with the apiserver. For further details, read the [Node Authorization](/docs/reference/access-authn-authz/node/).
+
+```kubelet.csr``` and ```proxier.csr``` need **hosts (SAN)** entries that match precisely the value of the node name provided by the kubelet.
```

Comment:
```suggestion
`kubelet.csr` and `proxier.csr` need **hosts (subjectAltName)** entries that match precisely the value of the node name provided by the kubelet.
```

BTW I'd spell out SAN because SAN has multiple meanings in IT, and this is terse.
To be honest, it's best to expand the term ahead of its first use on line 67. Even so, I'd use the expanded term, it doesn't take up much more space and it's more obvious what it means.

Reply:
Ok, i will expand the term SAN to subjectAltName